lib/board.rb

```diff
@@ -0,0 +1,56 @@
+require 'pry'
+class Board
+
+  attr_accessor :cells
+
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    #binding.pry
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(int)
+    int = int.to_i
+    cells[int-1]
+  end
+
+  def full?
+    cells.all? {|cell| cell != " "}
+  end
+
+  def turn_count
+    count = 0
+    cells.map { |cell|  count += 1 if cell != " "  }
+    count
+  end
+
+  def taken?(str)
```

Comment:
line 39 arguments should be the input passed to it. 
I would use this for the entire method 
```ruby
def taken?(input)
    position(input) == " "
end
```

the position method is already converting the input to an integer for us.